### PR TITLE
lzop: add page

### PR DESCRIPTION
**Pull request by KopfKrieg**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/lzop.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Compress a file into a new file with the .lzo suffix:
+
+`lzop {{file}}`
+
+- Decompress a file:
+
+`lzop -d {{file}}.lzo`
+
+- Compress a file, while specifing the compression level. 0 = Worst, 9 = Best (Default level is 3):
```
Comment (zdroid) [inline pages/common/lzop.md]:
```suggestion
- Compress a file, while specifing the compression level (0 being worst, 9 best, and 3 default):
```

Comment (KopfKrieg) [reply pages/common/lzop.md]: FYI: I copied this line from pages/zstd.md. If the line gets changed here we should maybe also change zstd's tldr-page.

Comment (zdroid) [reply pages/common/lzop.md]: Then we should change it in the both files. If you want, you can do it here as well.

**Comment by KopfKrieg** [comment]
Quick question: Do I need to apply the changes requested from zdroid in my master branch or can you just apply the changes without me doing anything?

**Comment by zdroid** [comment]
We can do it unrelated to this, I just wanted to take an opportunity in this PR.

**Pull request merged**